packages/RadioBoxGroup/src/RadioBoxGroup.tsx

```diff
@@ -1,15 +1,42 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import { emotion } from '@leafygreen-ui/lib';
-import RadioBox from './RadioBox';
+import RadioBox, { SizeValues, RadioBoxProps } from './RadioBox';
 
 const { css, cx } = emotion;
 
 const baseGroupStyle = css`
   display: flex;
 `;
 
-export default class RadioBoxGroup extends PureComponent {
+interface RadioBoxGroupProps {
+  children?: React.ReactNode;
+  onChange: React.ChangeEventHandler<HTMLInputElement>;
+  name?: string;
+  value?: string | number;
+  size: SizeValues;
+  className?: string;
+}
+
+interface RadioBoxGroupState {
+  value: string | number;
+}
+
+function isRadioBoxElement(
+  element: React.ReactNode,
+): element is React.ReactElement<RadioBoxProps, typeof RadioBox> {
+  return (
```

Comment:
I commented on this in the other PR as well – any reason we can't do the following?

```js
return element && element.type && element.type.displayName === 'RadioBox'
```

Reply:
Discussed on the other PR. Resolving.